libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.html

```diff
@@ -1,7 +1,7 @@
 <main *ngIf="challenge">
   <div id="bio">
     <h3 class="top-section-title">Overview</h3>
-    <p>{{ useNaIfEmpty(challenge.description) }}</p>
+    <p>{{ useNaIfFalsey(challenge.description) }}</p>
```

Comment:
Don't US speakers always adopt the shortest words? ;-)

> A falsy (sometimes written falsey) value is a value that is considered false when encountered in a [Boolean](https://developer.mozilla.org/en-US/docs/Glossary/Boolean) context.

Source: https://developer.mozilla.org/en-US/docs/Glossary/Falsy